packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -358,6 +365,13 @@ export default decorate([
                   type='text'
                   value={host}
                 />
+                {protocol === 'https' && (
+                  <span className='input-group-addon'>
+                    <Tooltip content={formatMessage(messages.remoteS3TooltipAcceptInsecure)}>
+                      <Toggle iconSize={1} onChange={effects.setAllowUnauthorized} value={allowUnauthorized} />
+                    </Tooltip>
+                  </span>
+                )}
```

Comment:
it really don't make sense with http protocol. Is there some similar case in xo ?

Reply:
I mean, in case of `https` protocol, if `allowUnauthorized` is an important information, it can be a good idea to give a direct view of this information.